test/E2E/TestFunctionApp/TestBasicHttpTriggerWithTriggerMetadata/run.ps1

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Input bindings are passed in via param block.
+param($req, $TriggerMetadata)
+
+# You can write to the Azure Functions log streams as you would in a normal PowerShell script.
+Write-Verbose "PowerShell HTTP trigger function processed a request." -Verbose
```

Comment:
Should we just generate this one?

Reply:
We could. That might be a good way to test our template.

Reply:
on second thought, that template actually tests grabbing stuff out of `$TriggerMetadata` so we can defer testing templates for now.